public/grpc-client.js

```diff
@@ -52,23 +52,26 @@ module.exports.init = async function({
   lndSettingsDir,
   network,
 }) {
+  const grpcOptions = {
+    keepCase: false,
+    longs: Number,
+    enums: String,
+    defaults: true,
+    oneofs: true,
+  };
   let credentials;
   let protoPath;
+  let atplProtoPath;
   let lnrpc;
   let unlocker;
   let lnd;
+  let autopilotRpc;
```

Comment:
Declare these two `let` vars in the event handler if they're only used there.

Reply:
removed this declaration. the other one is used elsewhere, so I left it.